cs3/storage/provider/v1beta1/resources.proto

```diff
@@ -135,29 +141,28 @@ enum LockType {
 // of this metadata according to their constraints, a reference
 // implementation is given here. The lock SHOULD be stored
 // as an extended attribute on the referenced filesystem entry.
-// It MUST NOT be accessible via the Stat/SetArbitraryMetadata APIs,
-// and it SHOULD contain a base64-encoded JSON with the following format:
-// {
-//   "type" : "<LOCK_TYPE>",
-//   "h" : "<holder>",
-//   "md" : "<metadata>",
-//   "mtime" : "<Unix timestamp>"
-// }
+// It MUST NOT be accessible via the Stat/SetArbitraryMetadata APIs.
```

Comment:
What about this? (sorry @butonic ...)
```suggestion
// Such extended attribute MUST NOT be exposed via the `Stat` and `SetArbitraryMetadata` APIs.
// Instead, the `ResourceInfo.Lock` attribute MUST be populated if a lock exists for the given reference.
```

Reply:
(or we drop this completely if it sounds too pedantic ;-) )

Reply:
No, it think we should guide implementations with the documentation.